packages/calypso-build/webpack/sass.js

```diff
@@ -38,6 +38,11 @@ module.exports.loader = ( { includePaths, prelude, postCssOptions, cacheDirector
 			loader: require.resolve( 'css-loader' ),
 			options: {
 				importLoaders: 2,
+				// We do not want css-loader to resolve absolute paths. We
+				// typically use `/` to indicate the start of the base URL,
+				// but starting with css-loader v4, it started trying to handle
+				// absolute paths itself.
+				url: ( path ) => ! path.startsWith( '/' ),
```

Comment:
`css-loader` was not previously parsing absolute paths, but it started to with v4. We are using `/` to denote things like `/calypso/img.png`, e.g. the webpack root, rather than an actual absolute path. Since the images don't exist at `/calypso` on the filesystem, the build fails. So to fix it, we need to enable the previous behavior, which we can do be not parsing absolute paths.